api/harborapi_test.go

```diff
@@ -137,7 +149,49 @@ func (a api) ProjectsPost(prjUsr usrInfo, project apilib.ProjectReq) (int, error
 
 	// body params
 	_sling = _sling.BodyJSON(project)
-	httpStatusCode, _, err := request(_sling, jsonAcceptHeader, prjUsr)
+	_, httpStatusCode, _, err := request(_sling, jsonAcceptHeader, prjUsr)
+	return httpStatusCode, err
+
+}
+
+//User Add
+func (a api) UserAdd(user apilib.User, authInfo ...usrInfo) (int, int, error) {
```

Comment:
//Creates a new user account.
//func (a api) UsersPost (user User) (error) {
//}
please follow standard function name, check all your function name. **include log and hot repo**
...
//Creates a new user account.
//func (a HarborApi) UsersPost (user User) (error) {
//}

//Mark a registered user as be removed.
//func (a HarborApi) UsersUserIdDelete (userId int32) (error) {
//}

//Update a registered user to change to be an administrator of Harbor.
//func (a HarborApi) UsersUserIdPut (userId int32) (error) {
//}